alphadia/workflow/peptidecentric.py

```diff
@@ -532,9 +567,12 @@ def calibration(self):
         )
 
         # Get the optimization lock
-        eg_idxes_for_calibration, precursor_df, fragments_df = (
-            self.get_optimization_lock()
-        )
+        (
+            self.optimization_lock_library_precursor_df,
```

Comment:
does this need to be a instance variable? I think we only need it within .calibration?

Reply:
It's needed in recalibration as well. I could pass it as an argument but I thought it was easier to have it as an instance variable.